jenkinsfiles/stable

```diff
@@ -56,31 +57,36 @@ pipeline {
                         values '9.0-jre11', '8.5-jre11'
                     }
                 }
+
                 environment {
                     DOCKER_IMAGE_NAME_BASE_FULL = "${DOCKER_IMAGE_NAME_BASE}:${DOCKER_IMAGE_TAG_BASE}"
-                    DOCKER_IMAGE_NAME_FULL = "${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG}-${DOCKER_IMAGE_TAG_BASE}" // used to test against different Tomcat variants
+                    DOCKER_IMAGE_NAME_FULL = "${DOCKER_IMAGE_NAME_DEV}:${DOCKER_IMAGE_TAG}-${DOCKER_IMAGE_TAG_BASE}" // used to test against different Tomcat variants
```

Comment:
I think we need some kind of slug (maybe the build number) in the image name to guard against concurrent builds overwriting each others images 🤔 Before this was not the case as we did not push/pull intermediary images.

Reply:
Do we really need this? The is the stable build which is invoked manually and not very often

Reply:
@radnov do you see any chance of what I described happening? If not you can mark this as resolved 😋

Reply:
Concurrent builds shouldn't really happen, as the stable pipeline is triggered manually by Phil for both branches (release candidates) and tags (releases). 👀